apps/api/v2/src/ee/bookings/2024-08-13/services/input.service.ts

```diff
@@ -89,7 +89,7 @@ export class InputBookingsService_2024_08_13 {
       ? await this.createBookingRequestOAuthClientParams(oAuthClientId)
       : DEFAULT_PLATFORM_PARAMS;
 
-    const location = request.body.meetingUrl;
+    const location = request.body.location || request.body.meetingUrl;
```

Comment:
event-type location and meetingUrl are of  two very different types right ? what happens if meetingUrl is just a string